packages/core/src/app/shipping/LeftToAllocateItemsTable.tsx

```diff
@@ -1,23 +1,28 @@
 import { FormikErrors } from "formik";
 import React, { FunctionComponent } from "react";
 
+import { TranslatedString } from '@bigcommerce/checkout/locale';
+
+import { isMobileView } from "../ui/responsive";
+
 import { AllocateItemsModalFormValues } from "./AllocateItemsModal";
 import LeftToAllocateItem from "./LeftToAllocateItem";
 import { MultiShippingTableItemWithType } from "./MultishippingV2Type";
-import { TranslatedString } from '@bigcommerce/checkout/locale';
 
 interface LeftToAllocateItemsTableProps {
     items: MultiShippingTableItemWithType[];
     formErrors: FormikErrors<AllocateItemsModalFormValues>;
 }
 
 const LeftToAllocateItemsTable: FunctionComponent<LeftToAllocateItemsTableProps> = ({ items, formErrors }: LeftToAllocateItemsTableProps) => {
+    const isMobileViewUI = isMobileView();
```

Comment:
Maybe just rename it to `isMobileView`

Reply:
I wanted to name it the same, but it conflicts with the `isMobileView` util method.

Reply:
Maybe we can rename ` isMobileView` method as `isMobileViewUI`.😂